plugins/inputs/ceph/README.md

```diff
@@ -46,15 +62,78 @@ Would be parsed into the following metrics, all of which would be tagged with co
 
   ## suffix used to identify socket files
   socket_suffix = "asok"
+
+  ## Ceph user to authenticate as
+  ceph_user = "client.admin"
```

Comment:
The original plugin poked at a socket, but that only gives basic data for a single process. The user here implicitly looks up a key (possibly via ceph.conf) which allows you to authenticate and talk to the entire cluster, all simple stuff for a ceph admin, but I'm anticipating you want less terse documentation?